apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx

```diff
@@ -116,14 +118,16 @@ export type ValidatorState = {
     };
 };
 
-function StakeColumn(prop: { stake: bigint; stakePercent: number }) {
+
+function StakeColumn({ stake, stakePercent }: { stake: bigint; stakePercent: number }) {
```

Comment:
Remove stakePercent?

Reply:
I think we may revert back to this later

Reply:
We can always add it back when we want, I'd rather the code reflect the current state of the app not potential future states

Reply:
We can keep the calculation in place, just remove this prop